Bottom: d5b2c10cbcd3cd9257f971d44ce48fc4f9166bf9
Top:    29780e89a1adaff725b75a2ec269c86abf6d6ed6
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-20 15:32:19 +0800

Refresh of autofs-5.0.7-fix-compile-error-with-heimdal-support-enabled.patch

---

diff --git a/modules/cyrus-sasl.c b/modules/cyrus-sasl.c
index b9c16f3..21bc00c 100644
--- a/modules/cyrus-sasl.c
+++ b/modules/cyrus-sasl.c
@@ -74,6 +74,7 @@ void _krb5_princ_realm(krb5_context context, krb5_const_principal princ,
 	else
 		*len = 0;
 	return;
+}
 #else
 void _krb5_princ_realm(krb5_context context, krb5_const_principal princ,
                           const char **realm, int *len)
